packages/grafana-ui/src/components/TransformersUI/CalculateFieldTransformerEditor.tsx

```diff
@@ -125,75 +122,55 @@ export class CalculateFieldTransformerEditor extends React.PureComponent<
     const { options } = this.props;
     const { names, selected } = this.state;
     const reducer = fieldReducers.get(options.reducer);
+
     return (
       <div>
-        <Label>Numeric Fields</Label>
-        <InlineList
-          items={names}
-          renderItem={(o, i) => {
-            return (
-              <span
-                className={css`
-                  margin-right: ${i === names.length - 1 ? '0' : '10px'};
-                `}
-              >
-                <FilterPill
-                  onClick={() => {
-                    this.onFieldToggle(o);
-                  }}
-                  label={o}
-                  selected={selected.indexOf(o) > -1}
-                />
-              </span>
-            );
-          }}
-        />
-        <Label>Calculation</Label>
-        <StatsPicker stats={[options.reducer]} onChange={this.onStatsChange} defaultStat={ReducerID.sum} />
-        <Label>Alias</Label>
-        <Input value={options.alias} placeholder={reducer.name} onChange={this.onAliasChanged} />
-
-        <Label>Replace all fields</Label>
-        <Switch checked={options.replaceFields} onChange={this.onToggleReplaceFields} />
-
-        {/*  nullValueMode?: NullValueMode; */}
+        <div className="gf-form-inline">
+          <div className="gf-form gf-form--grow">
+            <div className="gf-form-label width-8">Field name</div>
+            <HorizontalGroup spacing="xs">
```

Comment:
Would be better if the FilterPills used a gf-form-label class then we would not need this HorizontalGroup and the duplicated spacing param

Reply:
how come would it be better?:) I don't want  the new components to use any of "legacy" class names

Reply:
Because it's better to use our existing style and not duplicate it? the gf-form-label is using a bunch of variables so if we change the inline form styles we can modify them in one place I do now want many new components all mimicking & duplicating the inline form label style using custom space variables and colors directly.

Reply:
You are already using gf-form-label here and we want the item next to it to match the same background padding and spacing so why not use same class?

Reply:
But we do need a emotion-based component or emotion css mixing that does mimic gf-form-label but before that I think we also need to create some shared variables or strategy for sharing inline form styles in react / emotion.

Reply:
yes, totally agree. I'm gonna get back to ^^ ltr today